intl/messages/en.json

```diff
@@ -18,7 +18,12 @@
     "textDescription": "js-ipfs paves the way for the Browser implementation of the IPFS protocol. Written entirely in JavaScript, it **runs in a Browser, a Service Worker, a Web Extension and Node.js**, opening the door to a world of possibilities.",
     "currentVersion": "Current version:",
     "latestUpdate": "Latest update:",
-    "downloadsLastMonth": "Downloads last month:"
+    "downloadsLastMonth": "Downloads last month:",
+    "errorMessage": {
```

Comment:
Don't split a translation in multiple strings. This won't work well for arabic languages for instance. Instead use a template system, like https://lodash.com/docs/4.17.10#template with placeholders.

Reply:
I've changed it to one single error message in order to be simpler and not give unnecessary information to the user.

Reply:
Cool, but the issue persists for other translations as I mentioned. Could you create a new issue to fix those?